packages/webapp/pages/squads/[handle]/[token].tsx

```diff
@@ -128,39 +127,30 @@ const SquadReferral = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [member, trackedImpression]);
 
-  const sourceId = member?.source?.id;
   const { mutateAsync: onJoinSquad } = useMutation(
-    () => joinSquadInvitation({ sourceId, token }),
+    useJoinSquad({
+      squad: { handle, id: member?.source?.id },
+      referralToken: token,
+    }),
     {
       onSuccess: (data) => {
-        trackEvent({
-          event_name: AnalyticsEvent.CompleteJoiningSquad,
-          extra: joinSquadAnalyticsExtra(),
-        });
```

Comment:
Did we intentionally remove the tracking here?

Reply:
No it is moved to `useJoinSquad`